src/tools/wasm-reduce.cpp

```diff
@@ -1250,9 +1250,58 @@ int main(int argc, const char* argv[]) {
 
   const std::string WasmReduceOption = "wasm-reduce options";
 
-  ToolOptions options("wasm-reduce",
-                      "Reduce a wasm file to a smaller one that has the same "
-                      "behavior on a given command");
+  ToolOptions options(
+    "wasm-reduce",
+    R"(Reduce a wasm file to a smaller one with the same behavior on a given command.
+
+Typical usage:
+
+  wasm-reduce orig.wasm '--command=bash a.sh' --test t.wasm --working w w.wasm
```

Comment:
Is there an extra `w` here?
```suggestion
  wasm-reduce orig.wasm '--command=bash a.sh' --test t.wasm --working w.wasm
```